### DISCUSSION TITLE
Logging Devices

### DISCUSSION THREAD

**Discussion by Ignatious1** [Security-Onion-Solutions/securityonion#7617] [Unsupported Versions] [answered]
Hey all.

I am walking into a new environment and fairly new to SO. Is there anyway to find out what devices are sending logs to the SO?

**Comment by mnasec** [comment, marked as answer]
Just go into the Hunt Interface use the query:
`* | groupby "observer.name"`

This should give you the list of machines or nodes sending loginfo via different ways

**Reply by Ignatious1** [reply]
I'll give it a try. Thank you.
